spec/spec.md

```diff
@@ -598,7 +598,7 @@ where:
 
 - `payloads`: one or more base64-encoded transaction payload to sign
 - `options`: a JSON object, containing:
```

Comment:
I'm totally fine with this being non-optional, I just want to make sure that was on purpose.

Reply:
It can be optional - thanks, I'll fix.